runTests.py

```diff
@@ -33,26 +33,26 @@
 ]
 
 jumbo_folders = [
-    #"test/unit/math/prim/core",
+    "test/unit/math/prim/core",
     "test/unit/math/prim/err",
     "test/unit/math/prim/fun",
     "test/unit/math/prim/functor",
     "test/unit/math/prim/meta",
     "test/unit/math/prim/prob",
-    # "test/unit/math/rev/core",
+    "test/unit/math/rev/core",
     "test/unit/math/rev/err",
     "test/unit/math/rev/fun",
-    # "test/unit/math/rev/functor",
+    "test/unit/math/rev/functor",
     "test/unit/math/rev/meta",
     "test/unit/math/rev/prob",
-    # "test/unit/math/fwd/core",
+    "test/unit/math/fwd/core",
     "test/unit/math/fwd/fun",
-    # "test/unit/math/fwd/functor",
+    "test/unit/math/fwd/functor",
     "test/unit/math/fwd/meta",
     "test/unit/math/fwd/prob",
-    # "test/unit/math/mix/core",
+    "test/unit/math/mix/core",
     "test/unit/math/mix/fun",
-    # "test/unit/math/mix/functor",
+    "test/unit/math/mix/functor",
     "test/unit/math/mix/meta",
     "test/unit/math/mix/prob",
     "test/unit/math/opencl/device_functions",
```

Comment:
🚀 great stuff that you got all of them working!

Reply:
It was mostly a problem of needing to namespace code. If anything crops up in the future (easy to test because we can disable Jumbos for just one run now) it will most likely be something related to that as well (aka an easy fix, if you know what to look for)